lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -116,6 +119,18 @@ internal void CreateDefaultFolderStructure(string driverFolderPath)
         /// </summary>
         private void AddAssemblies()
         {
```

Comment:
ReSharper points out that several of the new vars introduced here aren't guaranteed to be different from `null`, but are passed to methods that assume them to be not `null`. Is this cause for concern?